include/deal.II/arborx/access_traits.h

```diff
@@ -383,9 +744,32 @@ namespace ArborX
     get(const std::vector<std::pair<dealii::Point<dim, Number>, Number>> &v,
         std::size_t                                                       i);
   };
+#  else
+  /**
+   * This struct allows ArborX to use std::vector<T> as primitive.
+   */
+  template <typename T>
+  struct AccessTraits<std::vector<T>>
+  {
+    using memory_space = Kokkos::HostSpace;
+
+    /**
+     * Return the size of the vector @p v.
+     */
+    static std::size_t
+    size(const std::vector<T> &v);
+
+    /**
+     * Return the ith element of the vector @p v.
+     */
+    static T
+    get(const std::vector<T> &v, std::size_t i);
+  };
```

Comment:
This will conflict with https://github.com/arborx/ArborX/pull/1262/files#diff-5cab312d233648f0cf7fab4a688ecc7b6f2460e20fa1287c0320f5243132db44 (or if someone else defined a specializarion for it). We should define our own `std::vector` wrapper like in https://github.com/arborx/ArborX/pull/1262/files#diff-576f6aef7391a0ca8107c0aa618490a78def4fc3532062260c639fc35c64d868 and not mess with namespaces we don't own.

Reply:
We won't merge that PR